example.py

```diff
@@ -42,3 +43,20 @@
 print('Team 4131\'s robots: ' + ', '.join('%s (%d)' % (robot.raw()['robot_name'], robot.raw()['year']) for robot in robots))
 print('Robots have attribute name:', 'name' in robots[0].raw())
 print('Robots have attribute robot_name:', 'robot_name' in robots[0].raw())
+print()
+
+print('-' * 8 + ' If-Modified-Since Header ' + '-' * 8)
+def fetch_cached_value():
+    return [tbapy.District({
```

Comment:
I don't see why you're manually initializing a district?

Reply:
This is just for the example. I was essentially trying to show how you could use a function that loaded a cached value in your code when working with the If-Modified-Since header.